docs/leaflet-draw-latest.html

```diff
@@ -138,6 +138,9 @@ <h3 id="adding-the-edit-toolbar">Adding the edit toolbar</h3>
 Leaflet.draw does not work with multigeometry features such as MultiPoint, MultiLineString, MultiPolygon,
 or GeometryCollection. If you need to add multigeometry features to the draw plugin, convert them to a
 FeatureCollection of non-multigeometries (Points, LineStrings, or Polygons).</p>
+<h3 id="draw-curves">Draw curves</h3>
+<p>In order to be able to draw and edit curves, the <a href="https://github.com/elfalem/Leaflet.curve">Leaflet.curve</a> plugin must be imported before Leaflet.draw.
+If it&#39;s not the case, the button to draw curves will simply not appear.<br>As the Renderer class must be defined to draw curves with L.curve, it is not enabled for Leaflet versions before 1.0.</p>
```

Comment:
This file is autogenerated by Leafdoc, so you should add docs directly to your sources.

Reply:
That's what I did. Should I not commit the generated docs ?